documents/api/viewsets.py

```diff
@@ -316,18 +324,23 @@ def partial_update(self, request, pk, *args, **kwargs):
             attachment_serializer = CreateAttachmentSerializer(data=data)
             attachment_serializer.is_valid(raise_exception=True)
             attachment_serializer.save()
-        # Don't update history if patch has no data.
-        if request.data:
+        # Update history only if status changed.
+        if request.data.get("status"):
```

Comment:
you could write here request.data.get("status") is not None:

Reply:
That is true. Would it matter?

Reply:
It could be clearer and more excplicit.